### DISCUSSION TITLE
Where should I store the CSS modules for components ? In components folder itself or in the styles folder

### DISCUSSION THREAD

**Discussion by lelouchB** [vercel/next.js#18295] [Help] [answered]
I am working on a blog project with Next.
Little confuse about where should I store the CSS modules for the components.

```
In components folder itself or in the styles folder ?
```

**Comment by jamesmosier** [comment, marked as answer]
Either way will work! I like to keep my CSS modules next to my component JS files, but I have worked on other projects that store them in a `styles` folder!

**Reply by lelouchB** [reply]
Thanks for the clarification.
